docs/antora.yml

```diff
@@ -0,0 +1,5 @@
+name: nile
+title: Nile
+version: 0.10.1
```

Comment:
not anymore! how should we keep up with it? adding it to RELEASING? to the CI?

Reply:
Good question! I would add it to the RELEASING guide along with the entry for updating the doc site (that is still missing). We could add a script (but to update just one file I feel is not worth it). Maybe add it to CI later?